common/djangoapps/student/views.py

```diff
@@ -1236,9 +1236,22 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
                 AUDIT_LOG.warning(u"Login failed - password for user.id: {0} is invalid".format(loggable_id))
             else:
                 AUDIT_LOG.warning(u"Login failed - password for {0} is invalid".format(email))
+
+        # Labster changes
+        # If there was login error, suggest student to login to UK region server
+        site_name = configuration_helpers.get_value('SITE_NAME', settings.SITE_NAME)
+        if not site_name == 'courses-uk.labster.com':
+            msg = _(
+                'If you are a student in the UK, your university may now be using the UK version of Labster. Please '
+                'try to log in here: <a href="https://courses-uk.labster.com">https://courses-uk.labster.com</a>'
+            )
+
+        else:
+            msg = _('Email or password is incorrect.')
+
         return JsonResponse({
             "success": False,
-            "value": _('Email or password is incorrect.'),
+            "value": msg,
```

Comment:
Who uses this message?

Reply:
@polesye when student fail to sign in ( email or password is incorrect/doesn't match) will show
this message,
If student sign in to Edx central,
![screenshot from 2017-09-12 17-38-35](https://user-images.githubusercontent.com/25189966/30323224-f164973e-97ee-11e7-93d6-47d5ef705dec.png)

If student sign in to Edx region,
![screenshot from 2017-09-12 18-30-29](https://user-images.githubusercontent.com/25189966/30323252-0a0a8b18-97ef-11e7-831f-5e72c9e9d1b9.png)